sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -4,9 +4,9 @@
 # Options to override the default behaviour of the FetchContent to include UR
 # source code.
 set(SYCL_PI_UR_OVERRIDE_FETCH_CONTENT_REPO
-  "" CACHE STRING "Override the Unified Runtime FetchContent repository")
+  "https://github.com/GeorgeWeb/unified-runtime.git" CACHE STRING "Override the Unified Runtime FetchContent repository")
 set(SYCL_PI_UR_OVERRIDE_FETCH_CONTENT_TAG
-  "" CACHE STRING "Override the Unified Runtime FetchContent tag")
+  "df1701b9addb75b58e241a86ad4ec828573707a4" CACHE STRING "Override the Unified Runtime FetchContent tag")
```

Comment:
These should be reverted.

Reply:
Yeah, gotta update that in general it's been stale for some time.